translib/transformer/xlate_del_to_db.go

```diff
@@ -139,14 +142,25 @@ func yangListDelData(xlateParams xlateToParams, dbDataMap *map[db.DBNum]map[stri
 		xlateParams.keyName = keyName
 	}
 
-	tblList, virtualTbl, err = tblKeyDataGet(xlateParams, dbDataMap, cdb)
-	if err != nil {
+	if xYangSpecMap[xlateParams.xpath].xfmrTbl != nil {
+		tblList, virtualTbl, err = tblKeyDataGet(xlateParams, dbDataMap, cdb)
+		if err != nil {
+			return err
+		}
+	} else if len(xlateParams.tableName) > 0 {
+		tblList = append(tblList, xlateParams.tableName)
+	}
+
+	if len(tblList) == 0 {
+		xfmrLogInfo("Unable to traverse list as no table information available at URI %v. Please check if table mapping available", xlateParams.uri)
 		return err
 	}
 
 	xfmrLogDebug("tblList(%v), tbl(%v), key(%v)  for URI (\"%v\")", tblList, xlateParams.tableName, xlateParams.keyName, xlateParams.uri)
 	for _, tbl := range tblList {
 		curDbDataMap, ferr := fillDbDataMapForTbl(xlateParams.uri, xlateParams.xpath, tbl, keyName, cdb, dbs, xlateParams.dbTblKeyGetCache, nil)
+		log.Info("********************************** ferr ", ferr, " curDbData ", curDbDataMap)
+		log.Info("*****************************  ", (*dbDataMap)[cdb])
```

Comment:
Do we need to log(info) the curDbDataMap, and dbDataMap?

Reply:
Addressed.